DigitalLearningSolutions.Data/Services/TrackerService.cs

```diff
@@ -63,6 +64,24 @@ public string ProcessQuery(TrackerEndpointQueryParams query)
             }
         }
 
+        private bool? ConvertToNullableBoolean(string? value)

```

Comment:
The way this converts strings to booleans is quite specific to tracker endpoint query params. Could be worth making that slightly more explicit in the title. E.g. Convert*Param*ToNullableBoolean.